docs/usage/configuration-options.md

```diff
@@ -1696,6 +1701,26 @@ Use this field to restrict rules to a particular language. e.g.
 }
 ```
 
+### matchCategories
+
+Use this field to restrict rules to a particular language or group. The categories can be found on the [manager documentation](./modules/manager.md). e.g.
+
+<!-- prettier-ignore -->
+!!! warning
+  Rules with `matchCategories` are only applied after extraction of a repository, therefore it can not be used to completely prevent analyzes of a repository.
+  Still it can be used to prevent or change the renovation of dependencies.
```

Comment:
```suggestion
<!-- prettier-ignore -->
!!! warning
    Rules with `matchCategories` are only applied after extraction of a repository, therefore it can not be used to completely prevent analyzes of a repository.
    Still it can be used to prevent or change the renovation of dependencies.
```

The admonition needs _four_ spaces here as well. 😉